cfme/utils/appliance/__init__.py

```diff
@@ -1697,6 +1707,19 @@ def server_roles(self, roles):
         self.set_yaml_config(yaml)
         wait_for(lambda: self.server_roles == roles, num_sec=300, delay=15)
 
+    def enable_embedded_ansible_role(self):
```

Comment:
**REQUIRED**
I don't think it's a good idea to have this in appliance. We already have [`server_roles_enabled`](https://github.com/ManageIQ/integration_tests/blob/master/cfme/configure/configuration/__init__.py#L1326) function.

Reply:
The reason for this is because whilst the other roles seem to start up very quickly, embedded ansible takes a LOT longer, it was thought unfair and potentially performance degrading to increase the wait for timeout, plus as it is a setter, getter, not very easy to do. We consulted with ronny too on this and we thought the helper function for ansible made sense.